src/eos-kolibri-manage.in

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+if [ "$EUID" -ne "0" ]; then
+  echo "$0 is not running as root. Try using sudo."
+  exit 2
+fi
+
+: ${KOLIBRI_HOME:="@KOLIBRI_DATA_DIR@"}
+
+KOLIBRI_MANAGE_CMD="@bindir@/flatpak run \
```

Comment:
This is, in general, wrong: Flatpak may be installed in a different prefix to eos-kolibri.

You can either use Meson's `find_program()` function to find the path to `flatpak` and hard-code that here. Or I think you can just write:

```suggestion
KOLIBRI_MANAGE_CMD="flatpak run \
```

and allow this to use `$PATH` in the normal way.